digits/scheduler.py

```diff
@@ -175,6 +177,30 @@ def add_job(self, job):
             return False
         else:
             self.jobs.append(job)
+
+            if flask._app_ctx_stack.top is not None:
```

Comment:
======================================================================
ERROR: digits.test_scheduler.TestSchedulerFlow.test_add_remove_job
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/jmancewicz/dev/digits/DIGITS/digits/test_scheduler.py", line 44, in test_add_remove_job
    assert self.s.add_job(job), 'failed to add job'
  File "/home/jmancewicz/dev/digits/DIGITS/digits/scheduler.py", line 183, in add_job
    html = flask.render_template('job_row.html', job = job)
  File "/usr/local/lib/python2.7/dist-packages/flask/templating.py", line 126, in render_template
    ctx.app.update_template_context(context)
AttributeError: 'NoneType' object has no attribute 'app'

I wasn't sure what was missing and it looks like I just committed the stopgap measure.  What's the correct way to avoid that error? @lukeyeager